src/test/java/persistence/study/ReflectionTest.java

```diff
@@ -0,0 +1,202 @@
+            () -> assertThat(methods.stream().map(Method::getName).toList()).containsExactly(
+                "printView"
+            )
+        );
+    }
+
+
+    @Test
+    @DisplayName("private field에 값 할당")
+    void privateField() {
+        // given
+        Car car = createCarWithDefaultConstructor();
+        final String NAME_TEST_VALUE = "test";
+        final int PRICE_TEST_VALUE = 111;
+
+        Field[] declaredFields = Car.class.getDeclaredFields();
+        List<Field> privateFields = Arrays.stream(declaredFields)
+                                          .filter(it -> it.getModifiers() == Modifier.PRIVATE)
+                                          .toList();
+        privateFields.forEach(
+            field -> {
+                field.setAccessible(true);
+                if (field.getType() == String.class) {
+                    fillStringArgument(car, field, NAME_TEST_VALUE);
+
+                } else if (field.getType() == int.class) {
+                    fillIntArgument(car, field, PRICE_TEST_VALUE);
+                }
+            }
+        );
+
+        assertAll(
+            () -> assertThat(getNameField(car)).isEqualTo(NAME_TEST_VALUE),
+            () -> assertThat(getPriceField(car)).isEqualTo(PRICE_TEST_VALUE)
+        );
+    }
+
+    @Test
+    void constructorWithArgs() {
+        Class<Car> carClass = Car.class;
+        final String NAME_TEST_VALUE = "test";
+        final int PRICE_TEST_VALUE = 111;
+        Constructor<?> constructorWithTwoParameter = Arrays.stream(carClass.getDeclaredConstructors())
+                                                           .filter(it -> it.getParameterCount() == 2)
+                                                           .findFirst().get();
+        Car car = createCarWithParameter((Constructor<Car>) constructorWithTwoParameter, NAME_TEST_VALUE,
+            PRICE_TEST_VALUE);
+
+        assertAll(
+            () -> assertThat(getNameField(car)).isEqualTo(NAME_TEST_VALUE),
+            () -> assertThat(getPriceField(car)).isEqualTo(PRICE_TEST_VALUE)
+        );
+    }
+
+    private Car createCarWithDefaultConstructor() {
+        Class<Car> carClass = Car.class;
+        try {
+            return carClass.getDeclaredConstructor().newInstance();
+        } catch (NoSuchMethodException | InstantiationException | IllegalAccessException |
+                 InvocationTargetException e) {
+            throw new RuntimeException("Create Car Fail");
+        }
+    }
+
+    private Car createCarWithParameter(Constructor<Car> constructor, String name, int price) {
+        try {
+            return constructor.newInstance(name, price);
+        } catch (InstantiationException | IllegalAccessException |
+                 InvocationTargetException e) {
+            throw new RuntimeException("Create Car Fail");
+        }
+    }
+
+    private void invokeMethod(Method method, Car car) {
+        try {
+            Object invoke = method.invoke(car);
```

Comment:
```suggestion
            method.invoke(car);
```

사용하지 않는 변수는 없애는게 좋다고 생각해요 🙏